docs/build-on-polkadot/xcm/hrmp-channels/overview.md

```diff
@@ -0,0 +1,211 @@
+---
+title: HRMP Channels
+description: HRMP channels enable cross-chain communication in Polkadot, a temporary solution before the more efficient XCMP protocol is implemented.
+---
+
+# HRMP Channels
+
+## Introduction
+
+Polkadot is designed to enable seamless interoperability between its connected parachains. At the core of this interoperability is the Cross-Consensus Message Format (XCM), a standard language that allows parachains to communicate and interact with each other.
+
+The actual network-layer protocol responsible for delivering XCM-formatted messages between parachains is called the Cross-Chain Message Passing (XCMP) protocol. XCMP maintains messaging queues on the relay chain, serving as a bridge to facilitate cross-chain interactions.
+
+However, as XCMP is still under development, Polkadot has implemented a temporary substitute known as Horizontal Relay-routed Message Passing (HRMP). HRMP provides the same interface and functionality as the planned XCMP, but with a key difference - it stores all messages directly in the relay chain's storage, which is more resource-intensive.
+
+Once XCMP is fully implemented, HRMP will be deprecated in favor of the native XCMP protocol. XCMP will offer a more efficient and scalable solution for cross-chain message passing, as it will not require the relay chain to store all the messages.
+
+## Prerequisites (WIP)
+
+<!-- TODO -->
+
+## Establishing HRMP Channels
+
+To facilitate communication between parachains using the HRMP protocol, the parachains must explicitly establish communication channels by registering them on the relay chain.
+
+Downward and upward channels from and to the relay chain are implicitly available, meaning they do not need to be explicitly opened.
+
+Opening an HRMP channel requires the parachains involved to make a deposit on the relay chain. This deposit covers the expenses of using the relay chain's storage for the message queues associated with the channel.
+
+### Relay Chain Parameters
+
+Each Polkadot relay chain has a set of configurable parameters that control the behavior of the message channels between parachains. These parameters include `hrmpSenderDeposit`, `hrmpRecipientDeposit`, `hrmpChannelMaxMessageSize`, `hrmpChannelMaxCapacity` and more.
+
+When a parachain wants to open a new channel, it must take these parameter values into account to ensure the channel is configured properly.
+
+To view the current values of these parameters in the Polkadot network:
+
+1. Visit [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpolkadot.api.onfinality.io%2Fpublic-ws#/explorer), navigate to the **Developer** dropdown and select the **Chain state** option
```

Comment:
In one of the tutorials I wrote, I've been requested to break down this into more granular actions. Maybe something like the following would work: 

```
1. Query Chain State
      a. Navigate to the **Developer** by using [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpolkadot.api.onfinality.io%2Fpublic-ws#/explorer)
      b. Select the **Chain state** option
```

However, this implies an update to the image to include the steps corresponding to those actions.